src/app/app.component.ts

```diff
@@ -3,6 +3,7 @@ import { Title } from '@angular/platform-browser';
 import { ActivatedRoute, Event, NavigationEnd, NavigationError, Router } from '@angular/router';
 import { Broadcaster, Logger } from 'ngx-base';
 import { BsModalService } from 'ngx-bootstrap/modal';
+import { ModalDirective } from 'ngx-bootstrap/modal';
```

Comment:
Why is this on a new line? It should be added to the import above.

Reply:
Because of auto import. I will fix this.